package/Dockerfile.dapper-base

```diff
@@ -39,5 +40,7 @@ RUN dnf -y distrosync && \
     curl -LO "https://github.com/kubernetes-sigs/kubefed/releases/download/v${KUBEFED_VERSION}/kubefedctl-${KUBEFED_VERSION}-linux-${ARCH}.tgz" && \
     tar -xzf kubefedctl-${KUBEFED_VERSION}-linux-${ARCH}.tgz && cp kubefedctl /usr/bin/ && chmod a+x /usr/bin/kubefedctl && \
     curl -Lo /usr/bin/kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-${ARCH}" && chmod a+x /usr/bin/kind && \
+    # FIXME: Use a release from GitHub once one is available
+    curl -Lo /usr/bin/armada https://drive.google.com/u/0/uc\?id\=17r6E0RwbHcvlaFl0LKFICuAc_eGb13W9\&export\=download && chmod a+x /usr/bin/armada && \
```

Comment:
I suggest we can take the binary from dima's repo [1] for now until it's merged in submariner-io

[1] https://github.com/dimaunx/armada/releases